app/src/main/java/com/mathroule/demoapp/Helper.java

```diff
@@ -0,0 +1,10 @@
+package com.mathroule.demoapp;
+
+import java.util.Random;
+
+public class Helper {
+
+    public static String getRandomButtonLabel( ) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/39d86fb759bdab1db76b818f1155db0332b36239/app/src/main/java/com/mathroule/demoapp/Helper.java#L7"></span>’)’ is preceded with whitespace.</td>
    </tr>
  </tbody>
</table>